tests/e2e/e2e_test.go

```diff
@@ -3,31 +3,58 @@ package e2e
 
 import (
 	"testing"
+
+	"github.com/llamastack/llama-stack-k8s-operator/api/v1alpha1"
 )
 
 func TestE2E(t *testing.T) {
 	registerSchemes()
 	// Run validation tests
 	t.Run("validation", TestValidationSuite)
 
-	// Track if creation tests passed
-	creationFailed := false
-
-	// Run creation tests
-	t.Run("creation", func(t *testing.T) {
-		TestCreationSuite(t)
-		creationFailed = t.Failed()
-	})
+	// Run combined creation and deletion tests for multiple distributions
+	// starter: newer image currently being actively updated
+	distributions := []string{"starter"}
```

Comment:
Would be nice to be able to detect the distro instead of having to change that code. Improvement for later I suppose.